### ISSUE TITLE
Provide low and high alternative constructor for Hyperrectangle

### ISSUE THREAD

**Issue by mforets** [JuliaReach/LazySets.jl#2]
In applications is sometimes useful to instantiate an Hyperrectangle as the pair of vectors `low` and `high`, instead of `center` and `radius` (both vectorial).

**Comment by schillic** [comment]
The problem here is that we cannot just write `Hyperrectangle(low, high)` because this constructor is already taken.

I see several options:
1. Add a dummy argument to the constructor. This is a good solution if the additional argument has a meaning, and otherwise (in my opinion) dumb. A compromise might be a flag `cr::Bool=true` (for **c**enter/**r**adius) which must be set to false for this alternative version.

2. Create a new subtype (e.g., `BHyperrectangle <: Hyperrectangle`, `B` for **b**ounds) which in the background calls the super constructor with modified values. Note that it automatically behaves like its parent. There might be minor performance penalties because then `Hyperrectangle` is not a base type anymore.

3. Create a function which looks like a constructor but is not (e.g., `BHyperrectangle` again) which just calls the real constructor with modified values. This is my current favorite.

**Comment by mforets** [comment]
i like your first option approach: to work out a bit the constructor such that it supports keyword arguments but also the current behavior, and finally all of the following work returning the same thing:

```julia
c = ones(2)
r = [0.1, 0.2]
l = [0.9, 0.8]
h = [1.1, 1.2]
H1 = Hyperrectangle(c, r)
H2 = Hyperrectangle(center=c, radius=r)
H3 = Hyperrectangle(low=l, high=h)
```

**Comment by schillic** [comment]
This is a good solution, I will do it.

**Comment by schillic** [comment]
We also added a function `norm` and improved the function `radius`.
Closed by merging #32.
